Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
I asked because I assumed that was the point of adding the protocol in the first place. If you're just interested in the caption view responding to `setPreferredMaxLayoutWidth:`, what's the point of the protocol?